src/Sylius/Bundle/CoreBundle/Resources/views/Email/passwordReset.html.twig

```diff
@@ -3,5 +3,5 @@
 {% endblock %}
 
 {% block body %}
-    {{ 'sylius.email.password_reset.content'|trans({'%username%': user.username, '%url%': url('sylius_shop_request_password_reset_token', { 'token' : user.passwordResetToken})})|raw }}
+    {{ 'sylius.email.password_reset.content'|trans({'%username%': user.username, '%url%': url('sylius_shop_password_reset', { 'token' : user.passwordResetToken})})|raw }}
```

Comment:
This file is placed in CoreBundle but it is coupled to shop-bundle. It should be moved out of here or a route should be declared both in core and shop bundle(with some other name)